appservice/scripts/websocket-session.py

```diff
@@ -12,21 +12,21 @@
 async def ws2out(ws):
     try:
         async for message in ws:
-            logger.debug('ws -> stdout: %s', message)
+            # logger.debug('ws -> stdout: %s', message)
```

Comment:
Move it back to `logger.debug` as the standard loglevel is now INFO?

Reply:
Dropped the calls to log the content.